src/Value/Information.php

```diff
@@ -26,48 +26,31 @@
  */
 class Information implements JsonSerializable
 {
-    /**
-     * @var BuildPath
-     */
-    private $buildPath;
-
-    /**
-     * @var string
-     */
-    private $environment;
-
-    /**
-     * @var bool
-     */
-    private $debuggerEnabled;
-  
-    /**
-     * @var array
-     */
-    private $systemInfo;
-    
-  
-    public static function buildFrom(BuildPath $buildPath, string $environment, bool $debuggerEnabled, array $systemInfo): Information
-    {
+    public static function buildFrom(
+        BuildInformation $buildPath,
```

Comment:
Beware of not initialising vars (see my article as well)

Reply:
I only want to set default values when they are usefull. In this case, all vars are initialized during construction time. None of the params are optional. So this should never lead to problems right?